data/json/bionics.json

```
@@ -167,12 +168,10 @@
     "name": { "str": "Subdermal Carbon Filament" },
     "description": "Lying just beneath your skin is a thin armor made of carbon nanotubes.  This reduces bashing damage by 2 and cutting damage by 4.",
     "occupied_bodyparts": [
-      [ "torso", 5 ],
+      [ "torso", 8 ],
```

```suggestion
      [ "torso", 6 ],
```
Judging relative to legs.